### PR TITLE
Doc: more readable document

### LINKED ISSUE #1755: If we need a code style?
**What would you like to be added**:

Good Code Style.

**Why is this needed**:
1. Standardized code promotes teamwork
2. Standardized code can reduce bug handling
3. Standardized code can reduce maintenance costs
4. Standardized code helps code review

**How**:
Although golang does not have a recognized code specification, we can refer to those:
https://go.dev/ref/spec
https://github.com/golang/go/wiki/CodeReviewComments
https://github.com/uber-go/guide/blob/master/style.md

e.g.

```go
var localIp string

func GetLocalIp() string {

}
```

into

```go
var localIP string

func GetLocalIP() string {

}
```

```go
type Service struct {
	name     string
	rcvr     reflect.Value
	rcvrType reflect.Type
	methods  map[string]*MethodType
}
```

into

```go
type Service struct {
	name    string
	rv      reflect.Value // receiveValue
	rt      reflect.Type  // receiveType
	methods map[string]*MethodType
}
```

### PR DESCRIPTION
**Pull request by pherzheyu**
**What this PR does**: 
1. Update some comment refers for https://dubbo-go.yuque.com/staff-ow02hm/hmg12h/isxeki.
2. Add some package comment for document extracting of godoc.
3. Fix some incorrect comment so the document is more readable.

**Which issue(s) this PR fixes**: 
Fixes #1755

**You should pay attention to items below to ensure your pr passes our ci test**
We do not merge pr with ci tests failed

- [x] All ut passed (run 'go test ./...' in project root)
- [x] After go-fmt ed , run 'go fmt project' using goland.
- [x] Golangci-lint passed, run 'sudo golangci-lint run' in project root.
- [x] After import formatted, (using [imports-formatter](https://github.com/dubbogo/tools#5-how-to-get-imports-formatter) to run 'imports-formatter .' in project root, to format your import blocks, mentioned in [CONTRIBUTING.md](https://github.com/apache/dubbo-go/blob/master/CONTRIBUTING.md) above) 
- [x] Your new-created file needs to have [apache license](https://raw.githubusercontent.com/dubbogo/resources/master/tools/license/license.txt) at the top, like other existed file does.
- [x] All integration test passed. You can run integration test locally (with docker env). Clone our [dubbo-go-samples](https://github.com/apache/dubbo-go-samples) project and replace the go.mod to your dubbo-go, and run 'sudo sh start_integration_test.sh' at root of samples project root. (M1 Slice is not Support)

### CODE REVIEW

File: common/url.go [hunk 1]
```diff
@@ -48,24 +48,17 @@ import (
 
 // dubbo role type constant
 const (
-	// CONSUMER is consumer role
 	CONSUMER = iota
-	// CONFIGURATOR is configurator role
 	CONFIGURATOR
-	// ROUTER is router role
```
Comment (AlexStocks) [inline common/url.go:55]: why delete such comments?

**Review by AlexStocks — APPROVED** [review]

**Review by zhaoyunxing92 — COMMENTED** [review]
filter/graceful_shutdown/doc.go  这个文件添加下l license

**Review by AlexStocks — APPROVED** [review]

File: common/constant/key.go [hunk 2]
```diff
@@ -291,62 +290,42 @@ const (
...
-	Tagkey      = "dubbo.tag"
-	// AttachmentKey in context in invoker
-	AttachmentKey       = DubboCtxKey("attachment")
-	TagRouterFactoryKey = "tag"
-	V3RouterFactoryKey  = "mesh"
+	TagRouterRuleSuffix       = ".tag-router"             // Specify tag router suffix
+	ConditionRouterRuleSuffix = ".condition-router"       // Specify condition router suffix
+	MeshRouteSuffix           = ".MESHAPPRULE"            // Specify mesh router suffix
+	ForceUseTag               = "dubbo.force.tag"         // the tag in attachment
+	Tagkey                    = "dubbo.tag"               //  key of tag
```
Comment (Lvnszn) [inline common/constant/key.go:297]: 多了个 空格

File: common/constant/version.go [hunk 3]
```diff
@@ -18,10 +18,7 @@
...
 const (
-	// Version apache/dubbo-go version
-	Version = "3.0.0"
-	// Name module name
-	Name = "dubbogo"
-	// Date release date
-	DATE = "2021/05/14"
+	Version = "3.0.0"      // apache/dubbo-go version
+	Name    = "dubbogo"    // module name
+	DATE    = "2021/05/14" //release date
```
Comment (Lvnszn) [inline common/constant/version.go:23]: 少了个空格

File: common/constant/time.go [hunk 4]
```diff
@@ -21,6 +21,6 @@ import (
 	"time"
 )
 
-// The value will be 10^6
+// MsToNanoRate  will be 10^6
```
Comment (Mulavar) [inline common/constant/time.go:24]: combine two lines into one?

File: common/rpc_service.go [hunk 5]
```diff
@@ -37,15 +37,14 @@ import (
 // RPCService the type alias of interface{}
 type RPCService = interface{}
 
-// ReferencedRPCService
-// rpc service interface
+// ReferencedRPCService is the rpc service interface which wraps base Reference method.
+//
+// Reference method refers rpc service id or reference id.
```
Comment (Mulavar) [inline common/rpc_service.go:42]: this line is a method comment, it should better be placed before the method Reerence()?

**Review by Mulavar — APPROVED** [review]

File: common/constant/time.go (hunk 4 above)
Comment (pherzheyu) [reply common/constant/time.go]: 已修改

File: common/constant/version.go (hunk 3 above)
Comment (pherzheyu) [reply common/constant/version.go]: 已修改

File: common/constant/key.go (hunk 2 above)
Comment (pherzheyu) [reply common/constant/key.go]: 已修改

File: common/rpc_service.go (hunk 5 above)
Comment (pherzheyu) [reply common/rpc_service.go:42]: image

It is also better to be consistent with the official interface code style.

**Review by cjphaha — APPROVED** [review]

**Pull request merged**